spec/support/vcr.rb

```diff
@@ -0,0 +1,17 @@
+require 'vcr'
+
+VCR.configure do |c|
+  c.cassette_library_dir  = File.expand_path('../../vcr', __FILE__)
+  c.hook_into :webmock
+end
+
+RSpec.configure do |c|
+  c.around(:each, :vcr) do |example|
+    name = underscorize(example.metadata[:full_description].split(/\s+/, 2).join("/")).gsub(/[^\w\/]+/, "_")
+    VCR.use_cassette(name) { example.call }
+  end
+end
+
+def underscorize(key) #:nodoc:
+  key.to_s.sub(/^(v[0-9]+|ns):/, "").gsub(/([A-Z]+)([A-Z][a-z])/,'\1_\2').gsub(/([a-z\d])([A-Z])/,'\1_\2').downcase
```

Comment:
Is it possible to combine the sub/gsub in one place so we can call `underscorize(example.metadata[:full_description])`?  Also, why do we need this? 😄 Thanks!

Reply:
We've temporarily 'borrowed' this vcr configuration from the fedex gem we're currently using. We'll have some future work dedicated to cleaning this up - but becuase the configuration we have in mealhand is too heavy to copy over - we're using this for the time being to ensure that we could get tests written.